.changes/next-release/feature-core-cced62e.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "core",
+    "contributor": "ziyanli-amazon",
+    "description": "As we decided to parse AwsQuery Compatible Error in the Http response header, the required changes need to take in place on both SDK client and service sides. This is the SDK client needed change.\nThis will be a pre-requisite for migrating services from AWSQuery wire protocol to AWSJson."
```

Comment:
Support  for clients with AWS Query protocol migrating towards AwsJson in order to  supports customizing error codes.